test/controllers/declaration/ProcedureCodeControllerSpec.scala

```diff
@@ -313,13 +314,11 @@ class ProcedureCodeControllerSpec extends ControllerSpec with ErrorHandlerMocks
       }
     }
 
-    val warehouseRequiredProcedureCodesSuffixes = Seq("07", "71", "78")
-    warehouseRequiredProcedureCodesSuffixes.foreach { suffix =>
-      s"user provides procedure code indicating WarehouseIdentifier is required (ends with $suffix" when {
+    warehouseRequiredProcedureCodes.foreach { suffix =>
```

Comment:
This test is suspicious. Is `suffix` used somewhere in the test code? The test actually does not seem to match the documentation.

Reply:
No, I can't see how these suffixes alter the test conditions either. I'd assume they were supposed to be concatenated with the procedureCode value on line 329, maybe give that a try and see if the tests still pass :thinking: